admin-dev/themes/new-theme/js/fos_js_routes.json

```diff
@@ -1 +1 @@
ds":["GET"],"schemes":[]},"admin_customers_search":{"tokens":[["text","\/sell\/customers\/search"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_customers_carts":{"tokens":[["text","\/carts"],["variable","\/","\\d+","customerId"],["text","\/sell\/customers"]],"defaults":[],"requirements":{"customerId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_customers_orders":{"tokens":[["text","\/orders"],["variable","\/","\\d+","customerId"],["text","\/sell\/customers"]],"defaults":[],"requirements":{"customerId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_carts_info":{"tokens":[["text","\/info"],["variable","\/","\\d+","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":{"cartId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]},"admin_carts_create":{"tokens":[["text","\/sell\/orders\/carts\/new"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_carts_edit_addresses":{"tokens":[["text","\/addresses"],["variable","\/","\\d+","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":{"cartId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_carts_edit_carrier":{"tokens":[["text","\/carrier"],["variable","\/","\\d+","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":{"cartId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_carts_set_free_shipping":{"tokens":[["text","\/rules\/free-shipping"],["variable","\/","\\d+","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":{"cartId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_carts_add_rule":{"tokens":[["text","\/rules\/add"],["variable","\/","[^\/]++","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_carts_delete_rule":{"tokens":[["text","\/delete"],["variable","\/","[^\/]++","cartRuleId"],["text","\/rules"],["variable","\/","[^\/]++","cartId"],["text","\/sell\/orders\/carts"]],"defaults":[],"requirements":[],"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_duplicate_cart":{"tokens":[["text","\/duplicate-cart"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_delete_product":{"tokens":[["text","\/delete"],["variable","\/","\\d+","orderDetailId"],["text","\/products"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+","orderDetailId":"\\d+"},"hosttokens":[],"methods":["POST"],"schemes":[]},"admin_orders_get_prices":{"tokens":[["text","\/prices"],["variable","\/","\\d+","orderId"],["text","\/sell\/orders\/orders"]],"defaults":[],"requirements":{"orderId":"\\d+"},"hosttokens":[],"methods":["GET"],"schemes":[]}},"prefix":"","host":"localhost","port":"","scheme":"http","locale":[]}
```

Comment:
@matks I've noticed weird config at the very end of this file 

> "host":"localhost","port":"","scheme":"http",

When running actual shop, it won't be on `localhost` and most likely it will be `https`, do you think current config may break things? :fearful:

Reply:
Crap. This is very likely 😢 we need to explore this and find out

Reply:
Then I guess you will have to add a js class to convert this config and inject the appropriate host and scheme?
Your local environment is always on `localhost` nobody uses another domain?

Reply:
Actually I'm on `prestashop.local` on my localhost and it works fine. :man_shrugging: